lib/tests/streamlit/runtime/scriptrunner/magic_test.py

```diff
@@ -142,7 +143,7 @@ def test_try_star_statement(self):
         self._testCode(CODE_TRY_STAR_STATEMENT, 9)
 
     @unittest.skipIf(
-        not magic.TRY_STAR_SYNTAX_AVAILABLE, "Not supported in this Python version"
+        not sys.version_info >= (3, 11), "Not supported in this Python version"
```

Comment:
Sorry the original code here was wrong where `TRY_STAR_SYNTAX_AVAILABLE` should be `MATCH_SYNTAX_AVAILABLE`, and now it should be `sys.version_info >= (3, 10)`.

Reply:
O, oki oki

I am fixing mypy errors and, and probably the best way it get rid of constants and do checks inline (mypy not smart enough to infer from constant, but could do that from an inline statement)